lib/presentation/ui/home/home_page.dart

```diff
@@ -177,9 +182,16 @@ class _MyHomePageState extends BaseState<HomePage> {
   /// Callback to navigate to Report a Problem screen
   void _onReportTap() {
     //todo: navigate to report a problem
-    
+
     Navigator.of(context).pushNamed(routeProblem);
   }
+
+  /// Callback to navigate to Report a Problem screen
```

Comment:
The method name is for occurrences but the comments talk about Report a Problem :P